include/faabric/transport/MessageEndpoint.h

```diff
@@ -72,13 +72,18 @@ class MessageEndpoint
   protected:
     const std::string host;
     const int port;
+    const std::string address;
```

Comment:
This address can be worked out at construction time so I made it a `const` member